pkg/jobrunaggregator/jobrunaggregatorlib/util.go

```diff
@@ -93,8 +94,15 @@ func WaitAndGetAllFinishedJobRuns(ctx context.Context,
 			finishedJobRunNames = append(finishedJobRunNames, jobRun.GetJobName()+jobRun.GetJobRunID())
 		}
 
-		summaryHTML := htmlForJobRuns(ctx, finishedJobRuns, unfinishedJobRuns)
-		if err := ioutil.WriteFile(filepath.Join(outputDir, "job-run-summary.html"), []byte(summaryHTML), 0644); err != nil {
+		summaryHTML := htmlForJobRuns(ctx, finishedJobRuns, unfinishedJobRuns, variantInfo)
+
+		var jobRunSummaryPrefix string
+		if len(variantInfo) == 0 {
+			jobRunSummaryPrefix = ""
+		} else {
+			jobRunSummaryPrefix = variantInfo + "-"
+		}
+		if err := ioutil.WriteFile(filepath.Join(outputDir, fmt.Sprintf("%sjob-run-summary.html", jobRunSummaryPrefix)), []byte(summaryHTML), 0644); err != nil {
```

Comment:
Any reason to use a file name over injecting a <title> tag into the HTML? I personally find it more readable than a file name

Reply:
I tried to put a `<title>/</title>` inside the `<head></head>` and it only changed the title of the tab and not the accordion menu.

Reply:
like this:

```
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
        <title>
            job-run-summary for vsphere-sdn-ipi
        </title>
    </head>
    <body>
        <div id="wrapper">
...
```

Reply:
I don't think editing it with web developer tools will work since it gets parsed by prow https://github.com/kubernetes/test-infra/pull/25409

Reply:
But I agree, if we can get the title to work, that'd be a lot simpler.